atomic_defi_design/qml/main.qml

```diff
@@ -42,12 +42,29 @@ DexWindow {
         border.width: 0
     }
 
-
-    App {
+    Loader {
+        id: app_loader
+        anchors.fill: parent
+        source: "App.qml"
+        function reload() {
+            source = ""
+            source = "App.qml?"+Math.random()
+        }
+    }
+    DexBusyIndicator {
+        anchors.centerIn: parent
+        running: app_loader.status === Loader.Loading
+        visible: running
+    }
+    Shortcut {
+        sequence: "Ctrl+L"
+        onActivated: app_loader.reload()
+    }
```

Comment:
Please try to add it only when hot reload is enable: `engine.rootContext()->setContextProperty("debug_bar", QVariant(true));` if i remember you have that when hot reload is enabled